components/org.wso2.carbon.identity.mgt.store.connector.jdbc/src/main/java/org/wso2/carbon/identity/mgt/store/connector/jdbc/connector/JDBCIdentityStoreConnector.java

```diff
@@ -518,6 +517,50 @@ public IdentityStoreConnectorConfig getIdentityStoreConfig() {
     }
 
     @Override
+    public List<String> getUsers(List<Attribute> attributes, int offset, int length)
+            throws IdentityStoreConnectorException {
+
+        List<String> userIdsToReturn = new ArrayList<>();
+        Map<String, String> properties = identityStoreConfig.getProperties();
```

Comment:
Can this identityStoreConfig be null? If yes, we need handle that possible NPE.

Reply:
Null check is done during the initialization by https://github.com/wso2-extensions/carbon-security-user-store-jdbc/pull/95

Reply:
it will be null if someone calls this method without calling init(IdentityStoreConfig)